src/inpage/sats.inpage.ts

```diff
@@ -30,7 +30,7 @@ const isValidEvent = (event: MessageEvent, method: SatsConnectMessageToContentSc
   return correctSource && correctMethod && !!data.payload;
 };
 
-const SatsMethodsProvider: BitcoinProvider = {
+const SatsMethodsProvider: Partial<BitcoinProvider> = {
```

Comment:
I'm curious what was the reasoning for Partial here? :)

Reply:
ah I see. I'd be keen to remove it before merge, since it was a good code hint before

Reply:
@m-aboelenein did you have a chance to follow up on this one?

Reply:
nice catch i reverted that change 👍